### PR TITLE
add more Info on Upgrade

### PR DESCRIPTION
**Pull request by jalogisch**
I notice that in the past month several ppl could not identify the way how to update Graylog. This little added text should give them the guidance they need.

FIX: https://github.com/Graylog2/documentation/issues/384

### CODE REVIEW

**Comment by jalogisch** [comment]
this need to be cherry-picked into 2.4 and 3.0 once merged

File: pages/upgrade.rst [hunk 1]
```diff
@@ -2,8 +2,19 @@
 Upgrading Graylog
 *****************
 
+When upgrading from a previous version of Graylog you basically follow the previous used installation method using the new version numbers.
```
Comment (kroepke) [inline pages/upgrade.rst]: "...you follow the previously used installation method..."

File: pages/upgrade.rst [hunk 2]
```diff
@@ -2,8 +2,19 @@
 Upgrading Graylog
 *****************
 
+When upgrading from a previous version of Graylog you basically follow the previous used installation method using the new version numbers. 
+
+The following Upgrade notes should be read carefully before you start the upgrade process. Breaking changes and dependency upgrades are documented in those upgrade notes.
+
+You should always follow the version trail and not jump a few versions up leaving out one or more versions. The upgrade notes are always written coming from the stable release before.
```
Comment (kroepke) [inline pages/upgrade.rst]: I'd write:
> You should always follow minor versions when updating across multiple versions to make sure necessary migrations are run correctly.

File: pages/upgrade.rst [hunk 3]
```diff
@@ -2,8 +2,19 @@
...
 .. toctree::
    :titlesonly:
    :glob:
 
    upgrade/graylog-*
+
+
+The Virtual Machine Appliance (OVA) and Amazon Web Services (AMI) Version of Graylog use the Omnibus package. The upgrade documentation is :ref:`part of the graylog-ctl documentation <upgrade_graylog_omnibus>`.
+
+Should the previous installation include the Graylog repository of the current Graylog version, update the repository package to the target version and use the system tools to upgrade the package. For .rpm based systems :ref:`this update guide <operating_package_upgrade_rpm-yum-dnf>` and for .deb based systems :ref:`this update guide <operating_package_upgrade_DEB-APT>` should help.
```
Comment (kroepke) [inline pages/upgrade.rst]: I would simplify this a bit:

> Should the current installation use operating system packages, update the repository package to the target version and use the system tools to upgrade the package.

**Review by kroepke — CHANGES_REQUESTED** [review]

File: pages/upgrade.rst (hunk 3 above)
Comment (jalogisch) [reply pages/upgrade.rst]: I had that before in such a simple way and notice that some users did not know how to do. Additional I want to link from that page to the actual page where the update is described.

**Review by kroepke — APPROVED** [review]

**Pull request merged**